zulipterminal/cli/run.py

```diff
@@ -135,6 +144,7 @@ def parse_zuliprc(zuliprc_str: str) -> Dict[str, Any]:
         if 'autohide' in config:
             settings['autohide'] = (config['autohide'], ZULIPRC_CONFIG)
 
+    print(settings)
```

Comment:
The settings contain the API key, which shouldn't be printed out in a screen.

Reply:
Yeah, sorry. As I said in another comment, this slipped my mind